index.html

```diff
@@ -1,9 +1,122 @@
-NPM Clone Nifty Penguin Magic npm Enterprise Products Solutions Resources Docs Support Search Join Log In Build amazing
-things Essential JavaScript development tools that help you go to market faster and build powerful applications using
-modern open source code. See plans Join for free Bring the best of open source to your company npm is the tool used by
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>npm</title>
+</head>
+<body>
+
+    <header class="header">
+
+        <div>
+
+            <div>
+                <img class="black-heart" src="images/black-heart.png" alt="black heart">
+                <span>Nifty Penguin Magic</span>
+            </div>
+
+          <nav>
+            <ul>
+                <li><a class="nav-bar-item" href="#">npm Enterprise</a></li>
+                <li><a class="nav-bar-item" href="#">Products</a></li>
+                <li><a class="nav-bar-item" href="#">Solutions</a></li>
+                <li><a class="nav-bar-item" href="#">Resources</a></li>
+                <li><a class="nav-bar-item" href="#">Docs</a></li>
+                <li><a class="nav-bar-item" href="#">Support</a></li>
+            </ul>
+          </nav>
+
+        </div>
+      
+        <div>
+            <img class="npm-logo" src="images/npm-logo.png" alt="npm logo">
+            <form id="search" method="GET" action="/search">
+                <input class="search-input" type="search" placeholder="Search packages">
+                <button class="search-button" type="submit">Search</button>
+            </form>
+            <div>
+                <a class="join-button" href="#">Join</a>
+                <a class="login-button" href="#">Log in</a>
+            </div>
+        </div>
+
+    </header>
+
+    <section id="first-section">
```

Comment:
Aqui poderia ser o Header da sua página, por ser a parte onde contem a idéia principal e os botoes call-to-action